pyega3/pyega3.py

```diff
@@ -15,7 +15,7 @@
 from pyega3.libs.data_file import DataFile
 from pyega3.libs.commands import execute_subcommand
 
-version = "3.4.2"
```

Comment:
Bumped the major version because I introduced a backwards incompatible change to the API by removing the --saveto parameter.